packages/round-manager/src/features/round/RoundDetailForm.tsx

```diff
@@ -205,19 +243,31 @@ export function RoundDetailForm(props: RoundDetailFormProps) {
             onSubmit={handleSubmit(next)}
             className="shadow-sm text-grey-500"
           >
+            {/* Round inputs */}
             <div className="pt-7 sm:px-6 bg-white">
-              <div className="grid grid-cols-6 gap-6">
+              <div className="grid grid-cols-6 gap-6 mb-4">
                 <RoundName
                   register={register("roundMetadata.name")}
                   errors={errors}
                 />
                 {program && <ProgramChain program={program} />}
               </div>
 
-              <p className="mt-6 mb-2 text-sm text-grey-400">
-                Where can applicants reach you and/or your team if support is
-                needed?
-              </p>
+              {/* vault Address */}
+              {roundCategory == RoundCategory.Direct && (
```

Comment:
`==` → `===`

Reply:
Will skip marking the remaining ones (if any) as by now you're probably getting sick of me doing it :)